### ISSUE TITLE
Feature Request: By default it might be better to open terminal in vertical split or horizontal split than in a tab.

### ISSUE THREAD

**Issue by debajyoti1990** [SpartanJ/ecode#68]
lapce editor for example does a very simple trick, ctrl+` opens terminal in hsplit and focuses on it. Hitting ctrl+` minimizes the terminal and so focus is back to editor now. I'm in windows so don't know MAC/Linux behavior.

**Comment by SpartanJ** [comment]
I generally create a terminal split too. There are keybindings for that already there:
I'm not sure about changing the default behavior, I find it a little confusing opening a split when not explicitly requested.
![Screenshot_20230304_130621](https://user-images.githubusercontent.com/650416/222916463-5088b198-181a-4216-8ca5-6e71d79d65b7.png)

Lapce takes a different approach than ecode. It's similar to most editors, the terminal area is separated from the document area, I think my approach is better, you can mix terminals and documents at will. But I'm open to suggestions regarding this topic.

**Comment by debajyoti1990** [comment]
The highlighted keybinding are little hard to follow if used as a daily driver. Moreover I just used Lapce's example just as a reference. I think VScode / Pycharm / Kate all have similar default behaviour what I suggested. If terminal is opened in a hsplit/vsplit it becomes easy to follow the code and run/repl at the same time. That's just my opinion though. If default behavior can't be changed, a drag and drop to hsplit/vsplit should also be good if implementable at all.

**Comment by SpartanJ** [comment]
The behavior will be configurable from the settings menu:
![new-terminal-behavior](https://user-images.githubusercontent.com/650416/223021024-3bf58f4f-670a-4373-a8d3-7a57f255c514.png)
The default configuration is the one visible on the screenshot.

**Comment by debajyoti1990** [comment]
Thanks.

**Comment by debajyoti1990** [comment]
Probably not worthy of a new issue, but, is there a way to setup same theme for file browser, terminal and editor?

**Comment by SpartanJ** [comment]
Those are separated components, the default theme for the three of them is the "same theme". I only ported a few themes for the editor and the terminal, they are really easy to port, but there's only one theme for the GUI (with dark and light versions). Currently, there are no alternatives, and changing the GUI theme isn't straightforward. I'll add an option to easily change the GUI theme, at least to override the colors, that's really easy actually (the theme is just a CSS file, and the colors are CSS variables). I want to provide the tools to easily customize the editor but I won't focus on adding themes and customizing the GUI, I think that's not a good use of my very limited time. And that's why I need collaborators =)
